servers/physics_2d/physics_server_2d_sw.h

```diff
@@ -288,6 +288,8 @@ class PhysicsServer2DSW : public PhysicsServer2D {
 	virtual void end_sync() override;
 	virtual void finish() override;
 
+	virtual void set_collision_iterations(int p_iterations);
```

Comment:
Missing `override` keyword (coding style check specific to master, that's why checks fail on some platforms)